README.md

```diff
@@ -30,6 +30,7 @@ PHP
 
 Python
 ---------
+- [Rhee - a general purpose Nepali programming language](https://github.com/biprasu/mrhee)
```

Comment:
Please follow the alphabetical ordering.

Reply:
Thanks for your comment. Please review the new pull request. Thanks.